fe/fe-core/src/main/java/com/starrocks/storagevolume/StorageVolume.java

```diff
@@ -292,6 +292,13 @@ public static Map<String, String> getParamsFromFileStoreInfo(FileStoreInfo fsInf
                     params.put(CloudConfigurationConstants.AWS_S3_NUM_PARTITIONED_PREFIX,
                             Integer.toString(s3FileStoreInfo.getNumPartitionedPrefix()));
                 }
+                if (s3FileStoreInfo.getPathStyleAccess() == 1) {
+                    params.put(CloudConfigurationConstants.AWS_S3_ENABLE_PATH_STYLE_ACCESS,
+                            Boolean.toString(true));
+                } else { // if 0 or 2, use virtual style
```

Comment:
shall put false only when `getPathStyleAccess() ==2`, it will be up to the CloudConfiguration's implementation details to determine what the default value is when the `aws.s3.enable_path_style` is not provided.